### DISCUSSION TITLE
Parameters tuning for Sequential Quadratic Programming

### DISCUSSION THREAD

**Discussion by roccafrancesco** [osqp/osqp#630] [General]
Hello 
i didn't find any support for the parameters tuning and my question is: 
if this algorithm were to be called by any Sequential quadratic Programming algorithm which, for particular problems, requires greater accuracy of the solution, how should the parameters (eps, alpha etc etc) be configured? Can we have an example with eps_SQP=10e-12?
Can someone please clarify this point or where can I find information on this?
thanks a lot.
Fra

**Comment by goulart-paul** [comment]
I think your question here is way too general to receive any kind of universally applicable answer.    This is probably more in the realm of "research question" than "github issue" I'm afraid.

**Comment by roccafrancesco** [comment]
Thank you for the fast reply.

P.S.  A Java version, based on paper:"An Operator Splitting Solver for Quadratic Programs(Bartolomeo Stellato, Goran Banjac, Paul Goulart, Alberto Bemporad, Stephen Boyd,February 13 2020)  is present on github at link: 
https://github.com/Hipparchus-Math/hipparchus/blob/master/hipparchus-optim/src/main/java/org/hipparchus/optim/nonlinear/vector/constrained/ADMMQPOptimizer.java 
The code  is first issue but thanks to it the development of a module for constrained nonlinear optimization has begun.
